manifests/init.pp

```diff
@@ -0,0 +1,69 @@
+# Class: lastpass
+#
+# This class installs the LastPass CLI and provides functions to interact with it.
+
+class lastpass (
+  $package  = $lastpass::params::package,
+  $home     = '/root/.lpass',
+  $agent_timeout = 3600,
+  $username = undef,
+  $password = undef,
+) inherits lastpass::params {
+
+  package { $package:
+    ensure => present,
+  }
+
+  file { '/usr/local/bin/lpasspw':
+    ensure => file,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+    source => "puppet:///modules/${module_name}/lpasspw",
+  }
+
+  file { $home:
+    ensure => directory,
+    mode   => '0600',
+  }
+  file { "${home}/pw":
+    ensure  => file,
+    mode    => '0400',
+    content => inline_template('<%= @password %>'),
+  }
+
+  profiled::script { 'lpass.sh':
```

Comment:
From the docs:

```
lpass stores configuration in the following locations, in descending order of precedence:

The directory $LPASS_HOME, if set

$XDG_CONFIG_HOME/lpass, $XDG_DATA_HOME/lpass, and $XDG_RUNTIME_DIR/lpass (or equivalent defaults), if at least $XDG_RUNTIME_DIR is set

$HOME/.lpass

```

My experience was an error if $LASTPASS_HOME was not set and the XDG variables were also not set. The lpass command appears to expect XDG paths to exist so it can check for the .lpass directory in them and never makes it to $HOME/.lpass. The intent seems correct, but the current behavior is not.

I'll poke around at this some more and see if a bug should be filed. Hopefully I can work around it for now.